packages/ignition/src/app/[lang]/page.tsx

```diff
@@ -18,12 +19,15 @@ export const generateMetadata = ({ params: { lang } }: PropsWithLangParams): Met
 };
 
 const Home = ({ params: { lang } }: PropsWithLangParams) => {
+  const { config } = withLocale(lang);
   const { "getting-started-slug": gettingStartedSlug, "getting-started": gettingStarted } =
-    withLocale(lang).config("nav");
+    config("nav");
+
+  const { organization, software } = withStructuredData(lang);
 
   return (
     <div className="flex flex-col grow overflow-y-auto items-center justify-between bg-cover bg-hero-light dark:bg-hero-dark">
-      <div className="relative w-full inset-0 before:absolute before:top-0 before:right-0 before:left-0 before:bottom-16 before:light-mask-image before:dark:dark-mask-image before:bg-grid before:bg-bottom before:bg-slate-900/[0.04] dark:before:bg-slate-400/[0.05] dark:before:border-b before:border-slate-200 dark:before:border-slate-100/5">
+      <main className="relative w-full inset-0 before:absolute before:top-0 before:right-0 before:left-0 before:bottom-16 before:light-mask-image before:dark:dark-mask-image before:bg-grid before:bg-bottom before:bg-slate-900/[0.04] dark:before:bg-slate-400/[0.05] dark:before:border-b before:border-slate-200 dark:before:border-slate-100/5">
```

Comment:
We can't have a `main` here once it's already on file `packages/ignition/src/app/[lang]/(content)/layout.tsx`